packages/kit/kit.vitest.config.js

```diff
@@ -12,8 +15,7 @@ export default defineConfig({
 		alias: {
 			'__sveltekit/paths': fileURLToPath(new URL('./test/mocks/path.js', import.meta.url))
 		},
-		// shave a couple seconds off the tests
-		isolate: false,
+		isolate: true,
```

Comment:
so much hate for this

Reply:
I think this is already the default so we could omit it https://vitest.dev/config/#isolate